dist/server/index.js

```diff
@@ -76,8 +92,15 @@ function afterLoaded(asset, { config: { tce } }) {
   const { sproutVideoApiKey: apiKey } = tce;
   const client = createClient({ apiKey });
   return client.videos.get(videoId)
-    .then(({ embed_code: embedCode }) => {
+    .then(video => {
+      const {
+        embed_code: embedCode,
+        selected_poster_frame_number: selectedPosterFrameIndex,
+        assets: { poster_frames: posterFrames }
```

Comment:
Consider encapsulating this transformations (from snake case to camelcase etc.) within the client.